### ISSUE TITLE
Dynamic route sometimes returns blank page

### ISSUE THREAD

**Issue by serge20** [serverless-nextjs/serverless-next.js#171]
Hi All,

I just deployed my nextjs app using serverless-next.js. I noticed an issue when i try to access a dynamic route. The first time it works fine, a couple of refresh it works and suddenly i get a blank/empty page with content-length:0, changing the dynamic query value makes it work again but same issue once u refresh a few times the blank/empty page with content-length:0 comes back again.

Has anyone ran into this? 

I used the example to setup the project and it seems that dynamic route support isn't working properly. any help would be much appreciated!

**Comment by danielcondemarin** [comment]
@serge20 Do you mean the DynamoDB example? https://github.com/danielcondemarin/serverless-next.js/tree/master/packages/serverless-nextjs-component/examples/dynamodb-crud

Also please share more information, a screenshot of all the HTTP Response and Request headers will help.

**Comment by serge20** [comment]
I followed the steps in the create-next-app example and added a dynamic route in the pages folder. 

I will add http response and request headers. Just a side note. I deployed the same project to zeit using now and I can't replicate the issue.

The blank page goes away when you change the query param so if test/1 went blank and i changed it to the /test/2 it works, but after a couple of refreshes you get the blank page again.  

Failing Response:
Link: https://ibb.co/bFdWShW

Successful Response:
https://ibb.co/WGyY2xD
